shm/Player.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <memory>
+#include "Cargo.hpp"
+#include "Ship.hpp"
```

Comment:
There should be newline between memory includes and other file includes.

Reply:
It's a matter of convention and if they don't want they don't need to stick to it.

Reply:
You should move to top headers